cmd/kubeadm/app/cmd/token.go

```diff
@@ -208,14 +208,18 @@ func NewCmdTokenGenerate(out io.Writer) *cobra.Command {
 }
 
 // RunCreateToken generates a new bootstrap token and stores it as a secret on the server.
-func RunCreateToken(out io.Writer, client clientset.Interface, cfgPath string, cfg *kubeadmapiv1beta2.InitConfiguration, printJoinCommand bool, kubeConfigFile string) error {
-	// KubernetesVersion is not used, but we set it explicitly to avoid the lookup
-	// of the version from the internet when executing LoadOrDefaultInitConfiguration
-	phaseutil.SetKubernetesVersion(&cfg.ClusterConfiguration)
+func RunCreateToken(out io.Writer, client clientset.Interface, cfgPath string, initCfg *kubeadmapiv1beta2.InitConfiguration, printJoinCommand bool, kubeConfigFile string) error {
+	// ClusterConfiguration is needed just for the call to LoadOrDefaultInitConfiguration
+	clusterCfg := &kubeadmapiv1beta2.ClusterConfiguration{
+		// KubernetesVersion is not used, but we set this explicitly to avoid
+		// the lookup of the version from the internet when executing LoadOrDefaultInitConfiguration
+		KubernetesVersion: constants.MinimumControlPlaneVersion.String(),
```

Comment:
possibly use this again here?
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/constants/constants.go#L387